src/app/[locale]/projects/detail/[id]/components/VulnerabilityTrackingStatus.tsx

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) Siemens AG, 2023. Part of the SW360 Frontend Project.
+
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+
+// SPDX-License-Identifier: EPL-2.0
+// License-Filename: LICENSE
+
+'use client'
+
+import { VulnerabilityTrackingStatus, Session, ProjectData, ProjectVulnerabilityTrackingStatus } from '@/object-types'
+import { SW360_API_URL } from '@/utils/env'
+import { useSession } from 'next-auth/react'
+import { Table, _ } from 'next-sw360'
+import Link from 'next/link'
+import { useTranslations } from 'next-intl'
+import { Spinner } from 'react-bootstrap'
+
+export default function VulnerabilityTrackingStatusComponent({ projectData }: { projectData: ProjectData }) {
+    const t = useTranslations('default')
+    const { data: session, status } = useSession()
+
+    const columns = [
+        {
+            id: 'vulnerabilityTrackingStatus.name',
+            name: t('Name'),
+            formatter: ({ id, name }: { id: string, name: string }) =>
+                _(
+                    <Link href={`/component/release/detail/${id}`} className='text-link text-center'>
+                        {name}
+                    </Link>
+                ),
+            sort: true,
+        },
+        {
+            id: 'vulnerabilityTrackingStatus.projectOrigin',
+            name: t('Project Origin'),
+            sort: true,
+        },
+        {
+            id: 'vulnerabilityTrackingStatus.SVMTrackingStatus',
+            name: t('SVM Tracking Status'),
+            sort: true,
+        },
+        {
+            id: 'vulnerabilityTrackingStatus.shortStatus',
+            name: t('Short Status'),
+            sort: true,
+        },
+        {
+            id: '   vulnerabilityTrackingStatus.type',
+            name: t('Type'),
+            sort: true,
+        }
+    ]
+
+    const initServerPaginationConfig = (session: Session) => { 
```

Comment:
Do not pass session, use the hook

Reply:
We are using hook only but passing on session like that to tackle the multiple rerendering of tables. PR #237